.travis.yml

```diff
@@ -46,14 +46,14 @@ d:
   - dmd-beta
 
 env:
-  - VIBED_DRIVER=libevent PARTS=lint,builds,unittests,examples,tests #,meson
-  - VIBED_DRIVER=vibe-core PARTS=builds,unittests,examples,tests
-  - VIBED_DRIVER=libasync PARTS=builds,unittests
+  - VIBED_DRIVER=libevent PARTS=lint,builds,unittests,examples,tests,mongo #,meson
+  - VIBED_DRIVER=vibe-core PARTS=builds,unittests,examples,tests,mongo
+  - VIBED_DRIVER=libasync PARTS=builds,unittests,mongo
```

Comment:
The separation is only for the Buildkite Project Tester and I am not even sure whether we run mongodb tests there.

Reply:
merged with master where these aren't there anymore